GNUmakefile

```diff
@@ -114,6 +114,10 @@ setup-testdb: ## Setup the test database.
 testdb: ## Prepares the test database.
 	go run . local db preparetest
 
+.PHONY: testdb
```

Comment:
We have 3 shadowed `.PHONY: testdb` - should they be unique?

Reply:
doesn't seem to matter, but i changed them to be unique